File: src/main/java/no/nav/fo/veilarbregistrering/besvarelse/UtdanningSvar.java
package no.nav.fo.veilarbregistrering.besvarelse;

import no.nav.fo.veilarbregistrering.metrics.Metric;

public enum UtdanningSvar implements Metric {
    INGEN_UTDANNING,
    GRUNNSKOLE,
    VIDEREGAENDE_GRUNNUTDANNING,
    VIDEREGAENDE_FAGBREV_SVENNEBREV,
    HOYERE_UTDANNING_1_TIL_4,
    HOYERE_UTDANNING_5_ELLER_MER,
    INGEN_SVAR;

    @Override
    public String fieldName() {
        return "utdanning";
    }

    @Override
    public String value() {
        return this.toString();
    }
}

File: src/main/java/no/nav/fo/veilarbregistrering/besvarelse/UtdanningSvar.kt
package no.nav.fo.veilarbregistrering.besvarelse

import no.nav.fo.veilarbregistrering.metrics.Metric

enum class UtdanningSvar : Metric {
    INGEN_UTDANNING,
    GRUNNSKOLE,
    VIDEREGAENDE_GRUNNUTDANNING,
    VIDEREGAENDE_FAGBREV_SVENNEBREV,
    HOYERE_UTDANNING_1_TIL_4,
    HOYERE_UTDANNING_5_ELLER_MER,
    INGEN_SVAR;

    override fun fieldName(): String = "utdanning"

    override fun value(): String = this.toString()
}